test/suites/dev/moonbase/test-moonbeam-lazy-migrations/test-unlock-democracy-funds.ts

```diff
@@ -0,0 +1,104 @@
+import "@moonbeam-network/api-augment";
+import { describeSuite, expect, beforeAll } from "@moonwall/cli";
+import { alith } from "@moonwall/util";
+import { ApiPromise } from "@polkadot/api";
+import { votingOf, locks } from "./first-100-votingof-and-locks-data.json";
+
+describeSuite({
+  id: "LM01",
+  title: "Lazy Migrations - Unlock Democracy Funds",
+  foundationMethods: "dev",
+  testCases: ({ context, it, log }) => {
+    let api: ApiPromise;
+
+    beforeAll(async () => {
+      api = context.polkadotJs();
+
+      for (let i = 0; i < votingOf.length; i++) {
+        await context.createBlock(
+          api.tx.sudo
+            .sudo(api.tx.system.setStorage([[votingOf[i]["key"], votingOf[i]["data"]]]))
+            .signAsync(alith)
+        );
+      }
```

Comment:
You could do this instead
```suggestion
await context.createBlock(
  api.tx.sudo
    .sudo(api.tx.system.setStorage(votingOf.map(v => [v.key, v.data])))
    .signAsync(alith)
);
```